packages/ODEs/README.md

```diff
@@ -0,0 +1,168 @@
+  * demonstrates the `output` feature
+
+Datagrok's ODEs suite has tools for solving both [stiff](https://en.wikipedia.org/wiki/Stiff_equation) and non-stiff equations. It provides a [numerical solution](https://en.wikipedia.org/wiki/Numerical_methods_for_ordinary_differential_equations).
+
+### Project structure
+
+A project defines [initial value problem](https://en.wikipedia.org/wiki/Initial_value_problem). It contains *name*, *differential equations*, *initial values* and *argument* specifications:
+
+```python
+#name: My ODEs
+#equations:
+  dx/dt = x + y + exp(t)
+  dy/dt = x - y - cos(t)
+#argument: t
+  initial = 0
+  final = 1
+  step = 0.01
+#inits:
+  x = 2
+  y = 5
+```
+
+You can create extensions by defining components in the control blocks.
+
+Find useful the following:
+
+|Control block|Features|
+|-------------|--------|
+|```#name```|Defines a name.|
+|```#equations```|Differential equation specification|
+|```#argument```|Independent variable specification|
+|```#inits```|Initial values specification|
+|```#constants```|Constants specification|
+|```#parameters```|Parameters specification|
+|```#expressions```|Defines auxiliary compuations.|
+|```#output```|Defines output columns and their captions|
+|```#tags```|Specifies tags (`model`, `app`, etc.).|
+|```#description```|Defines description of the model.|
+|```#meta.runOnOpen```|Provides computations immediately upon model launch after the project is exported to JavaScript script.|
+|```#meta.runOnInput```|Updates results immediately upon input changes after the project is exported to JavaScript script.|
+
+### Platform applications
+
+Datagrok provides an export of each solver project to JavaScript script:
+
+* Press **Export** button on the top panel
+* Press **SAVE** button
+
+Find the created script in the platform `Scripts`.
+
+Improve usability. Use `#tags: model` to add your model to `Model Catalog`. Provide a description in the `#description`-line:
+
+```python
+#name: Bioreaction
+#tags: model
+#description: Complex bioreaction simulation
+```
+
+Apply annotations in your project to get better UI. Append them, when defining *parameters*, *initial values* and *argument*.
+
+Group inputs by specifying their `category`:
+
+```python
+#parameters:
+  P1 = 1 {category: Parameters}
+  P2 = -1 {category: Parameters}
+```
+
+Add `units`:
+
+```python
+#inits:
+  x = 2 {units: C; category: Initial values}
+  y = 0 {units: C; category: Initial values}
+```
+
+Define the desired `caption`:
+
+```python
+#argument: t
+  start = 0 {caption: Initial; units: min; category: Time}
+  finish = 2 {caption: Final; units: min; category: Time}
+  step = 0.01 {caption: Initial; units: min; category: Time}
+```
+
+Provide hints in brackets `[ ]`:
+
+```python
+  P1 = 1 {category: Parameters} [P1 parameter tooltip]
+```
+
+The export feature provides an extension of your project with [scripting](https://datagrok.ai/help/compute/scripting) tools. You can add the desired functionality:
```

Comment:
This section needs to be rewritten, it's not clear what you mean here.

Reply:
I've replace it by:
`Apply scripting tools to get:

* non-elementary and special functions' use
* Datagrok packages' functions call`